test/on-error.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const t = require('tap')
+const Fastify = require('fastify')
+const From = require('..')
+const got = require('got')
+const httpStatus = require('http-status')
+const FakeTimers = require('@sinonjs/fake-timers')
+
+const clock = FakeTimers.createClock()
```

Comment:
What is the difference with the default setTimeout?
Reading the docs it seems the same to me with this usage

Reply:
the test runs faster, since the setTimeout doesn't wait the entire 1000 ms:
After the timeout error is returned from the target server, I fast-forward the setTimeout by using clock.tick